dspace-api/src/test/java/org/dspace/eperson/GroupTest.java

```diff
@@ -206,6 +206,9 @@ public void findAllNameSort() throws SQLException {
         List<String> names = new ArrayList<>();
         List<String> sortedNames = new ArrayList<>();
         for (Group group : groups) {
+            if (group.getName() == null) {
+                continue;
```

Comment:
@mwoodiupui : we agree, but are coming at it from different directions. Either groups should not be getting in without a name (this is the ideal situation), or we should have a way to handle unnamed groups.  Either way, I worry this unit test modification could be hiding a problem...why/how are unnamed groups getting in there? Maybe there's a separate Unit Test that is creating unnamed groups (and shouldn't be doing so)?